src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```

Comment:
While you did a great job working in the project requirements you submitted a projects that has no unit tests. its  says i  the projects requirements that you are supposed to write unit tests for  your  project. It would be nice if you implemented this.